frontend/src/components/accounts/Accounts.tsx

```diff
@@ -2,44 +2,32 @@ import React from "react";
 
 import AccountsApi, * as A from "../../libraries/explorer-wamp/accounts";
 
-import AccountsList from "./AccountsList";
+import autoRefreshHandler from "../utils/autoRefreshHandler";
 import PaginationSpinner from "../utils/PaginationSpinner";
+import AccountsList from "./AccountsList";
 
-export interface Props {}
-
-export interface State {
-  accounts: A.AccountBasicInfo[] | null;
+export interface Props {
+  Lists: A.AccountBasicInfo[];
```

Comment:
The library really make more generic but what HOC I create is good enough for our implementation. I know it has more function to use but we do not require them, right? I will definitely change the name.

Reply:
1. use inspiration means that we can take a look at their API design and learn what features they have:
    a. they pass an object, where the key is the props name (this will resolve `Lists` or `items`)
    b. they also pass information about the current state (pending / rejected / fulfilled), which helps to display more useful data
2. There is nothing bad about writing our own HOC if that makes sense. Ours is super tiny, but once we take (1.a) and (1.b) into account, it seems that their component is also of a reasonable size.